### ISSUE TITLE
The gamepad_battery config parameter is ignored

### ISSUE THREAD

**Issue by gwaub** [flightlessmango/MangoHud#761]
The `gamepad_battery` parameter is set to false by default internally but it's never checked for in hud_elements.cpp (or anywhere else) so changing it in the config file has no effect.

Adding a check like this in `HudElements::gamepad_battery()` just before calling `gamepad_update()` solves the issue:

```
void HudElements::gamepad_battery()
{
#ifdef __linux__
    if (HUDElements.params->enabled[OVERLAY_PARAM_ENABLED_gamepad_battery]) {
        gamepad_update();
...
```

And now gamepad batteries only show up when this parameter is explicitly enabled.

**Comment by gort818** [comment]
Oops, not sure how I forgot that.
Thanks for the report!

**Comment by gort818** [comment]
fixed with 9eb8447e14113e1583561c97d9fe67cd2b298f2f
